runtime/taskcontext.go

```diff
@@ -195,6 +212,15 @@ func (c *TaskContext) NewLogReader() (io.ReadCloser, error) {
 	return c.logStream.NextReader()
 }
 
+func (c *TaskContext) ExtractLog() (ioext.ReadSeekCloser, error) {
+	file, err := os.Open(c.logLocation)
```

Comment:
There is already an interface for reading the log... I don't think we should expose a seek interface..